atom/app/atom_main_args.cc

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2013 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "atom/app/atom_main_args.h"
+#include "base/strings/string16.h"
+#include "base/logging.h"
+
+namespace atom {
+  AtomCommandLine* AtomCommandLine::current_process_commandline_ = NULL;
+  AtomCommandLine::AtomCommandLine(NoProgram no_program) : argv_() {
+  }
+
+  AtomCommandLine::~AtomCommandLine() {
+  }
+
+  bool AtomCommandLine::Init(int argc,
+          const AtomCommandLine::CharType* const* argv) {
+    if (current_process_commandline_) {
+      return false;
+    }
+
+    current_process_commandline_ = new AtomCommandLine(NO_PROGRAM);
+    current_process_commandline_->InitFromArgv(argc, argv);
```

Comment:
We can make AtomCommandLine's data members `static` so we don't have to create a `current_process_commandline_` instance.